From fa67c7f9f70b24edfec27e6d2023a4aeb15ee8dd Mon Sep 17 00:00:00 2001
From: Gilles Dartiguelongue <eva@gentoo.org>
Date: Mon, 3 Aug 2009 00:06:15 +0200
Subject: [PATCH] Fix configure failure with --disable-dbus

AM_CONDITIONAL for HAVE_DBUS was badly placed resulting in configure
getting confused.

GNOME bug #590560
---
 configure.ac |   18 ++++++++++--------
 1 files changed, 10 insertions(+), 8 deletions(-)

diff --git a/configure.ac b/configure.ac
index d9cf17a..f06deab 100644
--- a/configure.ac
+++ b/configure.ac
@@ -166,15 +166,17 @@ AC_LANG_POP
 #
 # Use DBus to export our remote control if available.
 #
-AC_ARG_ENABLE(dbus, 
-	      [  --enable-dbus[[=no/yes]]    compile with dbus support [[default: yes]]],
-	      ENABLE_DBUS="$enableval", ENABLE_DBUS="yes")
-if test "x$ENABLE_DBUS" != "xno"; then
-        PKG_CHECK_MODULES(DBUS, dbus-c++-1 >= $DBUSCPP_VERSION, [
-              have_dbus="yes";AC_DEFINE([ENABLE_DBUS], [1], [Define to 1 if DBus is enabled])
-           ], have_dbus="no")
-        AM_CONDITIONAL(HAVE_DBUS, test "$have_dbus" = "yes")
+AC_ARG_ENABLE(dbus,
+	AS_HELP_STRING([--enable-dbus],[compile with dbus support @<:@default: yes@:>@]),
+	[],[enable_dbus=yes])
+
+if test "$enable_dbus" != "no"; then
+	PKG_CHECK_MODULES(DBUS, dbus-c++-1 >= $DBUSCPP_VERSION, [
+		have_dbus="yes"
+		AC_DEFINE([ENABLE_DBUS], [1], [Define to 1 if DBus is enabled])
+	], have_dbus="no")
 fi
+AM_CONDITIONAL(HAVE_DBUS, test "$have_dbus" = "yes")
 
 
 
-- 
1.6.3.3

